WordPress/Classes/Utility/ImmuTableViewController.swift

```diff
@@ -5,7 +5,6 @@ import WordPressShared
 typealias ImmuTableRowControllerGenerator = ImmuTableRow -> UIViewController
 
 protocol ImmuTablePresenter: class {
-    var visible: Observable<Bool> { get }
     func push(controllerGenerator: ImmuTableRowControllerGenerator) -> ImmuTableAction
```

Comment:
It's not being used yet, so I think I'll add it on the plans list changes, which need to present the details modally